### ISSUE TITLE
Cloning SVG loaded objects

### ISSUE THREAD

**Issue by dud3** [kangax/fabric.js#2598]
## Problem space
When cloning a `SVG loaded object` changes applied to the `parent object` will be applied to the `cloned object` as well.

E.x.: If fill-color of a path is applied to the `parent object` it gets applied to `cloned object` as well.

So the: `parent object` means the object that clone got created from.

## Example
Load the svg file:
```js
var loadSvg = function() {
  var url = 'path/to/file.svg';
  fabric.loadSVGFromURL(url, function (objects, options) {
    var obj = fabric.util.groupSVGElements(objects, options);
    var group = new fabric.Group(object.getObjects(), {originX:  'center', originY: 'center', centeredScaling: true, hasControls: true });
    canvas.add(group);
    canvas.renderAll();
  });
}
```

Simple clone function:
```js
var cloneObj = function()
  var aobj = canvas.getActiveOjbect();
  var clone = fabric.util["object"].clone(aobj);
  canvas.add(clone);
  canvas.renderAll();
}
```

## Note
One thing I've noticed is that:
The objects that got loaded from `loadSVGFromURL` get unique `svgUid`, I guess it increments from the last loaded SVG file.

When cloned the `svgUid` does NOT increment, it is the same as the `parent object svgUid`.

**Comment by dud3** [comment]
cc: @kangax @asturur

**Comment by asturur** [comment]
Please make a jsfiddle if you need help. it may be a custom code problem.
svgUid does not increment, if i remember well is used to bind svg and stylesheet while loading multiple svg togheter. After it is initialized, svgUid is useless ( i may be wrong ).

**Comment by dud3** [comment]
@asturur Just made a fiddle: http://jsfiddle.net/5KKQ2/431/

Thanks for the quick reply.

**Comment by asturur** [comment]
Ok is a bug, we are passing object reference and not clone of objects.

**Comment by dud3** [comment]
I've tried to clone the object, as: `$.extend({}, canvas.getActiveObject());`, but had no luck ;(.

Is there a work around to this?

I thought of converting it back to `SVG` and loading it again ?

I thought if turning

**Comment by asturur** [comment]
ok, everything fine!

http://jsfiddle.net/asturur/5KKQ2/433/

we have a big confusion about wich class is sync and async.
PathGroup is async. With new 2.0 and the use of promises everything will be sync.

PLease check fiddle for solution.

**Comment by dud3** [comment]
@asturur Great, check out what happens on `1.5.0`: http://jsfiddle.net/uyn2q3aa/

**Comment by asturur** [comment]
We do not have the numbers (in terms of team members)  to support older releases. please use just latest.
And also a fixed 1.5.0 is exactly the 1.6.0.

**Comment by dud3** [comment]
I see, yep that did the job, thanks.
